autopeering/selection/manager.go

```diff
@@ -266,16 +277,18 @@ func (m *manager) handleInRequest(req peeringRequest) (resp bool) {
 	if !m.isValidNeighbor(req.peer) {
 		return
 	}
-
-	reqDistance := peer.NewPeerDistance(m.getID().Bytes(), m.getPrivateSalt().GetBytes(), req.peer)
+	peerArs, _ := arrow.NewArRow(m.getArRow().GetExpiration().Sub(time.Now()), outboundNeighborSize, req.peer.Identity)
+	reqDistance := peer.NewPeerDistance(m.getArRow().GetRows()[req.channel], peerArs.GetArs()[req.channel], req.channel, req.peer)
 	filter := m.getConnectedFilter()
 	filteredList := filter.Apply([]peer.PeerDistance{reqDistance})
 	if len(filteredList) == 0 {
+		fmt.Print("Rejected because already connected to this node\n")
```

Comment:
The manager has a log that you should use instead of printing with fmt.
Something like:
```suggestion
		m.log.Debug("Rejected because already connected to this node")
```